Packages/MIES/MIES_MiesUtilities.ipf

```diff
@@ -6363,13 +6367,15 @@ threadsafe Function IsValidSweepAndConfig(sweep, config, [configVersion])
 		configVersion = DAQ_CONFIG_WAVE_VERSION
 	endif
 
+	if(!WaveExists(sweep))
+		return 0
+	endif
+
 	if(IsWaveRefWave(sweep))
 		return IsValidConfigWave(config, version = configVersion) &&  \
-				 IsValidSweepWave(sweep) &&                           \
```

Comment:
The call would additionally check if there is at least one channel in the sweep that can be resolved and contains data.
Should this really be omitted here?

Reply:
THanks, that was an oversight. Fixed.